src/tests/backend/specs/api/pad.js

```diff
@@ -586,6 +588,25 @@ describe(__filename, function () {
     });
   });
 
+  describe('appendText', function () {
```

Comment:
Mocha doesn't like duplicate names:

```suggestion
  describe('appendText again', function () {
```